functions/federated_aws_rp/static/index.js

```diff
@@ -26,6 +26,8 @@ const selectRole = async (e) => {
         method: "POST",
         body: JSON.stringify({
             arn: e.target.dataset.arn,
+            role: e.target.dataset.role,
+            alias: e.target.dataset.alias
```

Comment:
Alphabetize and use trailing commas.  :)

Reply:
Good idea, fixed in 313e6c7